modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney_controller.rb

```diff
@@ -45,17 +46,18 @@ def submit2122a
         end
 
         def validate2122
-          target_veteran
-          validate_json_schema
+          shared_form_validation('2122')
 
-          poa_code = form_attributes.dig('serviceOrganization', 'poaCode')
+          poa_code = get_poa_code('2122A')
```

Comment:
is this supposed to be 2122A? it's in a `validate2122` method and the shared validation gets `2122`?